js/navigation/NavigationLayout.ios.js

```diff
@@ -2,22 +2,93 @@
  * @providesModule NavigationLayout
  * @flow
  */
-
 import {
   StackNavigator,
   TabNavigator
 } from 'react-navigation'
-
 import SignInSignUpScreen from '../scenes/SignInSignUpScreen'
 import AccountScreen from '../scenes/AccountScreen'
 import MyPotLucksScreen from '../scenes/MyPotlucksScreen'
-import CreatePotluckScreen from '../scenes/CreatePotluckScreen'
+import InfoScreen from '../scenes/InfoScreen'
+import FoodScreen from '../scenes/FoodScreen'
+import InvitesScreen from '../scenes/InvitesScreen'
+import { colors, typography, windowWidth, windowHeight } from '../styles/baseStyles'
 
+const HostTabNavigator = TabNavigator(
+  {
+    Info: { screen: InfoScreen },
+    Food: { screen: FoodScreen },
+  },
+  {
+    initialRouteName: 'Info',
+    tabBarPosition: 'top',
+    tabBarOptions: {
+      activeTintColor: 'white',
+      inactiveTintColor: colors.mainBrandColor,
+      activeBackgroundColor: colors.mainBrandColor,
+      labelStyle: {
+        fontSize: typography.baseFontSize,
+        justifyContent: 'center',
+        paddingBottom: 3, //best option?
+        
+      },
+      tabStyle: {
+        borderRight: 1,
+      },
+      style: {
+        width: windowWidth*.9,
+        height: windowHeight*.04,
```

Comment:
maybe try having the height of this component be based on its content. i.e. the text and padding/margin of the text.